LayoutTests/imported/w3c/web-platform-tests/css/css-grid/parsing/grid-shorthand-serialization.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
```

Comment:
Ah, looks like my local wpt was out of date and only had two subtests here.

I had to change a few of the expected results, which hopefully are correct.

The first subtest was setting `grid: auto-flow auto / 100px 100px` and expecting `auto-flow auto / 100px 100px`, which I think fails the shortest serialisation principle, and `auto` is unnecessary. I changed it to accept `auto-flow / 100px 100px` or `none / 100px 100px` which seem equivalent, and I can't find a way to determine which should take precedence.

The third subtest was setting `grid: auto-flow / 10px; grid-template-rows: 20px` and expecting no serialisation for `grid`. I think since it's setting `grid-auto-flow: rows` (the default), we can still serialise to `20px / 10px`.